wiki/People/Beatmap_Nominators/id.md

```diff
@@ -0,0 +1,175 @@
+[false]: /wiki/shared/false.png "No"
+[true]: /wiki/shared/true.png "Yes"
+
+Beatmap Nominator
+==================
+
+*Lihat juga: [Peraturan Beatmap Nominator](/wiki/Beatmap_Nominator_Rules) dan [Informasi Umum untuk Beatmap Nominator](/wiki/Beatmap_Nominator_General_Information).*
+
+Beatmap Nominator, biasanya disebut sebagai *BN*, adalah kelompok yang bertugas bertanggung jawab atas [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure). Sejak 2014-08-22, mereka tidak lagi menjadi bagian dari staff, mencabut moderasi obrolan chat, nama merah di in-game, dan di dalam forum. Pada 2015-02-01, nama kelompok berubah dari *Beatmap Appreciation Team* menjadi *Beatmap Nominators*. Saat ini, mereka memiliki warna ungu pada nama mereka di forum.
```

Comment:
-biasanya disebut sebagai -> dikenal juga dengan.
-kelompok yang ~~bertugas~~ bertanggung jawab atas [penominasian beatmap].
-Sejak 2014-08-22, Beatmap Nominator telah dipisahkan dari bagian staf, perubahan ini meliputi pencabutan hak moderasi di chat in-game.